go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -3362,10 +3363,11 @@ func (fbo *folderBranchOps) statUsingFS(
 	}
 
 	if fi.Mode()&os.ModeSymlink != 0 {
-		sympath, err = fs.Readlink(name.Plaintext())
+		sympathPlain, err := fs.Readlink(name.Plaintext())
 		if err != nil {
 			return data.DirEntry{}, false, err
```

Comment:
might this error have a plaintext name in it?

Reply:
Not that I can tell.  If it does, it would be a bug in `libfs.FS`, or here in `folderBranchOps` which is used (recursively) by `libfs.FS`.  It's confusing.  But no, I don't think it can.